invenio_rdm_records/oaiserver/services/services.py

```diff
@@ -234,3 +236,15 @@ def read_all_formats(self, identity):
                 self, schema=self.config.metadata_format_schema
             ),
         )
+
+    def rebuild_index(self, identity):
+        """Rebuild OAI sets percolator index."""
+        entries = (
+            db.session.query(OAISet.spec, OAISet.search_pattern)
+            .yield_per(1_000)
```

Comment:
why are there two `yield_per` here?

Reply:
That's how one increases their yields 🤑  💰 📈 

I think I did a funky rebase and somehow got double lines, will remove :)